### PR TITLE
Add assignees column

### LINKED ISSUE #1: Display an "assigned" column

### PR DESCRIPTION
**Pull request by anandthakker**
Close #1

### CODE REVIEW

**Comment by anandthakker** [comment]
screen shot 2017-02-02 at 9 33 25 am

**Comment by anandthakker** [comment]
@lucaswoj rebased, feel free to use

**Comment by lucaswoj** [comment]
Just pulled in some changes via https://github.com/mapbox/top-issues/commit/5903aced1dcbad0c7654c214b2a0d9000abcc921

File: index.html [hunk 1]
```diff
@@ -64,7 +64,7 @@ <h1 id="title">Top Github Issues</h1>
 }
 
 function getParam(name) {
-    var match = RegExp('[?&]' + name + '=([^&]*)').exec(window.location.search);
+    var match = RegExp('[?&]' + name + '=([^&/]*)').exec(window.location.search);
```
Comment (lucaswoj) [inline index.html:67]: What prompted this change?

File: index.html [hunk 2]
```diff
@@ -136,6 +136,25 @@ <h1 id="title">Top Github Issues</h1>
...
+function getAssigneesSortKey(issue) {
+    const assignees = issue.assignees.sort().map(assignee => assignee.login.toLowerCase());
+    if (assignees.length === 0) {
+        return '\ue83a'; // force unassigned to the bottom of the sort
+    }
+    if (githubUsername && assignees.indexOf(githubUsername.toLowerCase()) >= 0) {
+        return '!'; // force user's own issues to top of the sort
+    }
+    return assignees.join(',');
+}
```
Comment (lucaswoj) [inline index.html:148]: I'm erring on the side of not including this sorting logic. We may soon move away from users providing their usernames and personal access tokens.

File: index.html (hunk 1 above)
Comment (anandthakker) [reply index.html:67]: Huh.  When I was working on it before, I was getting an automatically-added trailing slash that messed up this parsing, but now I can't reproduce it.

**Pull request closed without merging**